### ISSUE TITLE
Market at "wrong" place in building menu

### ISSUE THREAD

**Issue by bunnybot** [widelands/widelands#6842]
[<img align='left' src='https://codeberg.org/avatars/80fed3275155709856459727e9311437' alt='MisterW' height='48' />](https://codeberg.org/MisterW)*[Mirrored from Codeberg](https://codeberg.org/wl/widelands/issues/5201)*
*Created on Tue Jul 08 10:47:03 CEST 2025 by [**MisterW**](https://codeberg.org/MisterW)*

---

### Describe the Bug

If you click on a big building space and then if you click by memory on an icon, the wrong building gets selected - because the market is at the beginning on the list, so all positions are changed.

### To Reproduce

-

### Expected Behavior

If you click on a big building space, the market should be at the end of the list, not at the beginning. So no building changes its position at the list.

### Operating System

-

### Widelands Version

1.3~git1 (378aef0@master) Release

### Enabled Add-Ons

-

### Additional Context

-

**Comment by bunnybot** [comment]
[<img align='left' src='https://codeberg.org/avatars/a4ad2f9a2504f052397d8ba82ed48ba1' alt='Nordfriese' height='48' />](https://codeberg.org/Nordfriese)*[Mirrored from Codeberg](https://codeberg.org/wl/widelands/issues/5201#issuecomment-5754632)*
*On Tue Jul 08 11:08:18 CEST 2025, [**Benedikt Straub (Nordfriese)**](https://codeberg.org/Nordfriese) wrote:*

---

While the changed positions will take some getting used to, I believe that the Market should logically be positioned at the beginning of the list in the same way the Warehouse is also in the first position of its list, due to the relative importance of this building. The order is now Warehouses – Markets – Productionsites – Trainingsites – Militarysites which represents a logical smooth progression from economy-focused to military-focused building types. Grouping markets with military buildings would be illogical IMO.

**Comment by bunnybot** [comment]
[<img align='left' src='https://codeberg.org/avatars/a4ad2f9a2504f052397d8ba82ed48ba1' alt='Nordfriese' height='48' />](https://codeberg.org/Nordfriese)*[Mirrored from Codeberg](https://codeberg.org/wl/widelands/issues/5201#issuecomment-6837256)*
*On Sun Aug 31 23:49:21 CEST 2025, [**Benedikt Straub (Nordfriese)**](https://codeberg.org/Nordfriese) wrote:*

---

There seems to be consensus not to change the order.
